software/proposals/neighbourhood_assistance.md

```diff
@@ -9,9 +9,10 @@ title: "Neighbourhood Assistance"
 | Tags           |      |
 | Created Date   |      |
 | Author         | Doug |
-| Leads          | Jao Faria, Lucia Carreira |
+| Leads          | Joao Faria, Lucia Carreira, Pedro Filipe, Shayan Chowdhury |
 | Slack Channel  |      |
```

Comment:
`#neighborhood-assistance` here right?

PS: we should also rename the channel to `#project-neighborhood-assistance` according to the guidelines I guess

Reply:
At the moment few site-admins can do change slack settings and the queue is long. I would keep it as it is if you want to be found, and update it right away when the situation changes.